db/migrate/20211014143555_convert_date_time_in_newbookings.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: false
+
+# Migration to convert the datetime column into a date and time column in newbookings table
+class ConvertDateTimeInNewbookings < ActiveRecord::Migration[6.1]
+  # Temporary class to hold newbooking data while converting
+  class Newbooking20211014143555 < ApplicationRecord
+    self.table_name = 'newbookings'
+
+    belongs_to :pet
+    belongs_to :service
+  end
+
+  def change
+    newbooking = Newbooking20211014143555.all
+    newbooking.each do |booking|
+      begin
+        date = booking.session.to_date
+        time = booking.session.to_time
+        booking.update!(date: date, time: time)
+      rescue StandardError
```

Comment:
This is really useful for debugging an issue... but in a production setting, you would want the error to still be raised so that the migration gets stopped (and somebody knows to go and look at the problem).

You can do this by either not rescuing the exception, or if the debug information is useful, by rescuing and re-raising the exception after you've reported it.

Code like this will re-raise:
```ruby
begin
  raise 'time to explode...'
rescue StandardError => e
  puts 'it exploded'
  raise e
end
```